neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -24,18 +24,49 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation: str='RELU',
```

Comment:
Já bych byl pro to použít callable všude, kde to jenom rozumně jde. `GRUCell` a `LSTMCell` nejsou callables stejného typu (konstruktory mají argumenty) a potom se podle použité buňky ještě přepíná, co půjde jako query state do attentionu.